mutations/events.py

```diff
@@ -586,7 +594,13 @@ def progressEvent(
     elif (
         updated_event_instance.status.state == Event_State_Status.pending_room
     ):
-        cc_to = getRoleEmails("cc")
+        cc_to = getRoleEmails("cc") + (
+            [
+                mail_club,
+            ]
```

Comment:
why are we ccing the student body as body will be for slo

Reply:
To let the student body also know about what's going on, and keep track of the event approval status. Moreover, this ccing was introduced for clubs in the first place so that in case of email failure, or if they want to follow up for the same email, they can reply to the email. If not CCed, they won't have the option to do so.